app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -229,8 +232,13 @@ export default {
     },
   },
 
-  mounted() {
-    this.fetchForm(this.formId);
+  async mounted() {
+    // Get the form and latest form fields
+    await this.fetchForm(this.formId);
+    await this.fetchFormFields({
+      formId: this.formId,
+      formVersionId: this.form.versions[0].id,
+    });
```

Comment:
Does this need to be async/awaited? I'm not seeing an ordering dependency between fetchForm and fetchFormFields (unless they're doing something to the vuex state as a side effect and need to stall on each other)

Reply:
I looked for a way to avoid this, but it was the only method I saw to get the latest form fields for the comparison shortly thereafter. I may not have to await fetchFormFields, but that call is dependent on having the form first.